### ISSUE TITLE
missing nested key fails

### ISSUE THREAD

**Issue by sethzollinger** [i18next/i18next#1332]
Hi, i am building a proof of concept and wasn't using any resource files or backend. I was Trying to nest keys which fails if they are not found, i was expecting the nested key being returned as a fallback value. Is there a workaround for this or can this behavior be changed?
Repro:
```javascript
const i18next = require('i18next') // "i18next": "^17.0.16"
i18next.init({
    fallbackLng: 'en',
    ns: ['Common'],
    defaultNS: 'Common'
    //,resources: { en: { Common: { Foo: 'Foo' } } }
})
console.log(i18next.t('Common:Foo'))
console.log(i18next.t('Common:Bar', '{{val}}Bar', { val: '$t(Common:Foo)' }))
```
Output:
```bash
> node index.js

Foo
/node_modules/i18next/dist/cjs/i18next.js:810
      return this.resourceStore.getResource(code, ns, key, options);
                                ^

RangeError: Maximum call stack size exceeded
    at Translator.getResource (/node_modules/i18next/dist/cjs/i18next.js:810:33)
    at /node_modules/i18next/dist/cjs/i18next.js:786:32
    at Array.forEach (<anonymous>)
    at /node_modules/i18next/dist/cjs/i18next.js:760:17
    at Array.forEach (<anonymous>)
    at /node_modules/i18next/dist/cjs/i18next.js:757:20
    at Array.forEach (<anonymous>)
    at Translator.resolve (/node_modules/i18next/dist/cjs/i18next.js:745:12)
    at Translator.translate (/node_modules/i18next/dist/cjs/i18next.js:574:27)
    at /node_modules/i18next/dist/cjs/i18next.js:715:35
```
Thanks

**Comment by jamuhl** [comment]
Just my guess...https://github.com/i18next/i18next/blob/master/src/Interpolator.js#L174 is the problem...we will have to assert defaultValue gets removed to avoid such a loop

**Comment by sethzollinger** [comment]
thanks for your fast response, for the time being i am just defining the required keys

**Comment by jamuhl** [comment]
should be fixed in i18next@17.0.17
